flipperdevices/flipperzero-firmware
# SubGHz - Keeloq: Read Centurion Nova remotes

# What's new

- Added support for decoding Centurion Nova remotes (keeloq)

P.S. Thanks Carlos for help

# Verification 

- Check Reading of other keeloq remotes that they still work fine
- Try decoding this recording to see that its now decoded properly, not as MF: Unknown
[RAW_20230718-014032.sub.zip](https://github.com/flipperdevices/flipperzero-firmware/files/12074259/RAW_20230718-014032.sub.zip)


# Checklist (For Reviewer)
